code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -302,6 +302,19 @@
 	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
 	new /obj/item/weapon/reagent_containers/pill/tramadol(src)
 
+/obj/item/weapon/storage/pill_bottle/paracetamol
+	name = "bottle of Paracetamol pills"
+	desc = "Contains pills used to relieve pain but better than Tramadol!!."
```

Comment:
Consider losing the "!." to make it grammatically correct, a period after an exclamation point is awkward and two exclamation points seems unnecessary.